src/clientside/config.cpp

```diff
@@ -8,10 +8,15 @@ ActuatorMessage build_command_message() {
   return ActuatorMessage{
       .ov101 = digitalRead(pinout::MISSILE_SWITCH_1),
       .ov102 = digitalRead(pinout::MISSILE_SWITCH_2),
-      .nv101 = digitalRead(pinout::MISSILE_SWITCH_3),
-      .ov201 = digitalRead(pinout::MISSILE_SWITCH_4),
-      .ov202 = digitalRead(pinout::MISSILE_SWITCH_5),
-      .injector_valve = digitalRead(pinout::MISSILE_SWITCH_INJECTOR),
+      .nv201 = digitalRead(pinout::MISSILE_SWITCH_3),
+      .nv202 = digitalRead(pinout::MISSILE_SWITCH_4),
+	  .vent_valve = digitalRead(pinout::MISSILE_SWITCH_5),
+	  .injector_valve = digitalRead(pinout::MISSILE_SWITCH_INJECTOR),
+	  .fill_dump_valve = digitalRead(pinout::MISSILE_SWITCH_6),
+      .fill_disconnect = digitalRead(pinout::MISSILE_SWITCH_7),
+	  .rocket_power = digitalRead(pinout::MISSILE_SWITCH_IGNITION_SEC), // Out of switch, should use omnibus dashboard
```

Comment:
If we're going to control rocket power via dashboard (seems reasonable) then remove this line?